code/modules/hydroponics/grown.dm

```diff
@@ -26,49 +26,48 @@
 		plantname = planttype
 
 /obj/item/weapon/reagent_containers/food/snacks/grown/Initialize()
-	..()
-	spawn()
-		if(!plantname)
-			return
+	. = ..()
+	if(!plantname)
+		return
 
```

Comment:
These lines below checking for plant controller can be deleted since it will surely be there